
'''

[예외 처리]

예외 발생할때 프로그램 런타임 탈출 못하도록 
방어해주는 방법! 


try:
   (필수) 예외 발생 가능한 명령어
except: 
   (필수) 예외 발생할때 실행할 명령어
else: 
    (옵셔널 - 순서 준수) 예외 안발생했을때 실행할 영역
finally:
    (옵셔널 - 순서 준수) 예외 발생여부
    상관없이 수행하는 영역

'''

arr = ['2','4','6']
try:
    print( arr[3])
except: 
    print('error')
